### DISCUSSION TITLE
Home Offset Issues

### DISCUSSION THREAD

**Discussion by JoeMaxey** [mriscoc/Ender3V2S1#45] [Q&A] [answered]
Running DD on my E3v2 with 422 board. Upgraded to the recent FW version and am having trouble setting my home offsets. I used Pronterface to enter the X/Y offsets (M206 X7 Y11), saved those (M500), and even set the home offsets in the Menu to match those (X7 Y11) I set in the FW. None of those seem to work though as the printer is still offset (e.g., waste line is off the bed and the bed leveling print is not centered, forward and to the left as you look at the bed). What am I missing?

Also, I used this technique (M206/M500) to set the home offsets in the previous FW version and it worked flawlessly.

**Comment by mriscoc** [comment, marked as answer]
The home offsets move the printable area but not the probing area, to move both you must use the X MIN and Y MIN. 
https://github.com/mriscoc/Ender3V2S1/wiki/How-to-set-the-home-offsets

**Reply by JoeMaxey** [reply]
Thank you, Miguel! So I've been testing different settings in the meantime and may have figured it out. First, I set all offsets (in Cura and FW) to 0. Then I printed a quick bed leveling file with five circles, one circle in the middle of the bed. I measured the offset of the center circle to each bed side in the X and Y direction. In the printer menu, I changed the X and Y home offsets to half the differences in the X and Y directions (X10.5 and Y13.5) and printed the file again. This moved the center circle in the wrong direction. I went back into the printer menu and set the home offsets to X-10.5 and Y-13.5. That put the waste line on the bed along with the bed leveling print. I'm reprinting that now to measure my center circle to more perfectly center the bed re the print.
